docs/source/en/quicktour.mdx

```diff
@@ -134,8 +134,7 @@ Audio files are automatically loaded and resampled when calling the `"audio"` co
 Let's extract the raw waveform arrays of the first 4 samples and pass it as a list to the pipeline:
 
 ```py
->>> raw_audio_waveforms = [d["array"] for d in dataset[:4]["audio"]]
->>> speech_recognizer(raw_audio_waveforms)
+>>> speech_recognizer(dataset[:4]["audio"])
```

Comment:
Nice! Could you update the `es` version as well?

Reply:
Done !